internal/controller/controller.go

```diff
@@ -20,6 +20,10 @@ import (
 	"go.opentelemetry.io/ebpf-profiler/util"
 )
 
+const (
+	MiB = 1 << 20
+)
```

Comment:
nit:

```suggestion
const MiB = 1 << 20
```
unless you see more consts coming up